docs-ref-services/preview/messaging-eventgrid-readme.md

```diff
@@ -401,17 +400,18 @@ byte[] dataInBytes = eventData.toBytes();
 ```  
 
 #### Deserialize system event data from `CloudEvent` or `EventGridEvent`
-An event that is sent to a [System Topic](https://docs.microsoft.com/azure/event-grid/system-topics) is called a
+An event that is sent to a [System Topic](/azure/event-grid/system-topics) is called a
```

Comment:
looks like there is break between line 403 and 404

Reply:
Nice catch!